app/admin/crm/[id]/page.tsx

```diff
@@ -24,6 +28,21 @@ export default async function ContactPage({
 
   const contactData = contact[0];
 
+  const chatSessions = await db
```

Comment:
This is incorrect. We should be using Stripe to determine whether or not this contact is a paying customer of Vargas JR's Salary